Product/Production/Common/CONNECTCoreLib/pom.xml

```diff
@@ -28,7 +28,7 @@
             <groupId>org.connectopensource</groupId>
             <artifactId>CONNECTCommonTypesLib</artifactId>
         </dependency>
-
+        
```

Comment:
When modifying an XML, make sure to do two things - trim trailing space, and pretty print formatting.